src/components/utils/file.js

```diff
@@ -1,16 +1,23 @@
 import loadImage from 'blueimp-load-image/js/load-image'
 import {canvasToBase64Images, toLossyImageDataUrl} from './canvas.js'
 
-export const fileToBase64 = (file, callback, errorCallback) => {
+export const convertFile = (file, callback, errorCallback) => {
   const reader = new FileReader();
-  reader.readAsDataURL(file);
-  reader.onload = function () {
-   callback(reader.result)
+  // If PDF return base64. For other images return blob
```

Comment:
We are converting PDFs to base64 in order to send them to the server through websockets as the implementation only accepts a string. We won't need to do this conversion when we will use the HTTP protocol and PDFs will be treated just as files. 
We are returning a Blob for every capture, being it a PDF, a File or a screenshot from the webcam. Blob and File are very similar because File objects inherit from Blob. We couldn't return File because the file constructor is not supported on IE11. So the client will be handling all the captures in the same way